hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -172,4 +174,13 @@ void Exception(int errorCode, String className, @Nullable String message, StackT
     @Response(ResponseMessageConst.ENTRIES_WITH_CURSOR)
     void EntriesWithCursor(int tableIndex, List<Map.Entry<Data, Data>> entries);
 
+    @Response(ResponseMessageConst.SCHEDULED_TASK_HANDLER)
+    void ScheduledTaskHandler(ScheduledTaskHandler handler);
+
+    @Response(ResponseMessageConst.SCHEDULED_TASK_STATISTICS)
+    void ScheduledTaskStatistics(ScheduledTaskStatistics stats);
```

Comment:
You defined custom codecs and did a single parameter of that custom type here but what we normally did up to now is NOT define custom codec but just encode the parameters here and let it auto-generate, can you look at the examples in this file?